src/TestFramework/TestSubscriber.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Omnipay\Vindicia\TestFramework;
+
+use Guzzle\Common\Event;
+use PaymentGatewayLogger\Event\Constants;
+use PaymentGatewayLogger\Event\ErrorEvent;
+use PaymentGatewayLogger\Event\RequestEvent;
+use PaymentGatewayLogger\Event\ResponseEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class TestSubscriber implements EventSubscriberInterface
+{
+    const PRIORITY = 0;
+
+    /** @var array */
+    public $eventsDispatched = array();
+
+    /**
+     * Returns an array of event names this subscriber wants to listen to.
+     *
+     * The array keys are event names and the value can be:
+     *
+     *  * The method name to call (priority defaults to 0)
+     *  * An array composed of the method name to call and the priority
+     *  * An array of arrays composed of the method names to call and respective
+     *    priorities, or 0 if unset
+     *
+     * For instance:
+     *
+     *  * array('eventName' => 'methodName')
+     *  * array('eventName' => array('methodName', $priority))
+     *  * array('eventName' => array(array('methodName1', $priority), array('methodName2')))
+     *
+     * @return array The event names to listen to
+     */
+    public static function getSubscribedEvents()
```

Comment:
if you extended https://github.com/vimeo/payment-gateway-logger/blob/master/src/Event/Subscriber/OmnipayGatewayRequestSubscriber.php, you wouldn't need to define this function